Fix ID3D12GraphicsCommandList::ResolveSubresource

john-h-k opened the pull request.

john-h-k committed:
Fix ID3D12GraphicsCommandList::ResolveSubresource

PRMerger10 commented:
@john-h-k : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edit pass

Stripping out `en-US` so we can support non-en-US locales. Also replacing all absolute urls with site-rel, in order to support Help Viewer. And then changing all out-of-date `/windows/desktop/` to the correct `/windows/win32/` to save the redirector some work and to help out folks reading the source who're unaware of the change. Also various other cleanup, but I stopped short of replacing all the HTML with markdown.
